### DISCUSSION TITLE
Copilot in GitHub.com now available for subscribers of Copilot Individual or Business plans

### DISCUSSION THREAD

**Discussion by novium2** [community/community#139874] [Copilot Conversations]
With a subscription to Copilot Individual or Copilot Business, you can now access Copilot in GitHub.com, allowing you to:

- Discover codebases on GitHub effortlessly using powerful natural language code search using [Copilot Chat](https://docs.github.com/en/enterprise-cloud@latest/copilot/using-github-copilot/asking-github-copilot-questions-in-githubcom).
- Streamline development processes by receiving suggestions to resolve build failures and [summarizing changes](https://docs.github.com/en/enterprise-cloud@latest/copilot/using-github-copilot/creating-a-pull-request-summary-with-github-copilot) in pull requests.
- Quickly get up to speed with the help of Copilot through summaries and key takeaways from [discussions, issues, pull requests and more](https://docs.github.com/en/enterprise-cloud@latest/copilot/using-github-copilot/asking-github-copilot-questions-in-githubcom#powered-by-skills).
- These features are also now available in GitHub Mobile for all Copilot users.
- If you’re enrolled into our [recently announced o1 model limited beta](https://github.blog/changelog/2024-09-19-sign-up-for-openai-o1-access-on-github/), you can experiment with o1-preview and o1-mini directly in GitHub.com. To gain access to o1, please visit the [waitlist](https://github.com/o1-waitlist-signup).

[Read more here.](https://github.blog/changelog/2024-09-26-copilot-in-github-com-now-available-on-copilot-individual-and-copilot-business-public-preview/)

**Comment by jamie6king** [comment]
The article says "bottom-left of the interface" and proceeds to go to the bottom-right lol.

**Comment by BOEF16** [comment]
> With a subscription to Copilot Individual or Copilot Business, you can now access Copilot in GitHub.com, allowing you to:
> 
> - Discover codebases on GitHub effortlessly using powerful natural language code search using [Copilot Chat](https://docs.github.com/en/enterprise-cloud@latest/copilot/using-github-copilot/asking-github-copilot-questions-in-githubcom).
> - Streamline development processes by receiving suggestions to resolve build failures and [summarizing changes](https://docs.github.com/en/enterprise-cloud@latest/copilot/using-github-copilot/creating-a-pull-request-summary-with-github-copilot) in pull requests.
> - Quickly get up to speed with the help of Copilot through summaries and key takeaways from [discussions, issues, pull requests and more](https://docs.github.com/en/enterprise-cloud@latest/copilot/using-github-copilot/asking-github-copilot-questions-in-githubcom#powered-by-skills).
> - These features are also now available in GitHub Mobile for all Copilot users.
> - If you’re enrolled into our [recently announced o1 model limited beta](https://github.blog/changelog/2024-09-19-sign-up-for-openai-o1-access-on-github/), you can experiment with o1-preview and o1-mini directly in GitHub.com. To gain access to o1, please visit the [waitlist](https://github.com/o1-waitlist-signup).
> 
> [Read more here.](https://github.blog/changelog/2024-09-26-copilot-in-github-com-now-available-on-copilot-individual-and-copilot-business-public-preview/)

**Comment by makskustomer** [comment]
I toggled this on for my org and it sent a welcome to github copilot enterprise email to my users even though we are on copilot business. We were previously on a github copilot enterprise trial and are back on business now.

**Comment by kbdharun** [comment]
Hi, It would be great if we could publically share conversations/chats with others similar to ChatGPT it can either be sharing the direct chat links or in GitHub case fine-grained sharing with a specific set of user(s) or team(s) or people from an org.

**Comment by nakamoto-masa** [comment]
I have activated Copilot individual from iPad today.
And I have confirmed that Copilot individual is active in my profile in github.com.
But no copilot icon is shown in github.com, and a recommendation to start trial is shown instead.
What is wrong?

My profile: Copilot individual is activated.
![image](https://github.com/user-attachments/assets/09421ecb-e984-4cf5-94da-9eb2f29aa22d)

My repository: copilot icon is not shown.
![image](https://github.com/user-attachments/assets/09ab66f5-6af7-4648-a650-91d403291432)
